vector/src/main/java/im/vector/app/features/start/StartAppViewModel.kt

```diff
@@ -55,11 +55,15 @@ class StartAppViewModel @AssistedInject constructor(
         handleLongProcessing()
         viewModelScope.launch(Dispatchers.IO) {
```

Comment:
Can you take care of inject Dispatchers instead of using directly IO?

Reply:
will do :+1:

Reply:
https://github.com/vector-im/element-android/pull/6710/commits/3725921400631977c5fac483b18d25d10da84347